kernel/os/src/os_mbuf.c

```diff
@@ -1036,3 +1036,90 @@ os_mbuf_widen(struct os_mbuf *om, uint16_t off, uint16_t len)
 
     return 0;
 }
+
+struct os_mbuf *
+os_mbuf_pack_chains(struct os_mbuf *m1, struct os_mbuf *m2)
+{
+    uint16_t rem_len;
+    uint16_t copylen;
```

Comment:
slight in-consequence in naming length rem_len has _ while copylen does not

Reply:
Yes; I am not terribly consistent. I just thought remlen looked weird whereas copylen looks nice :-)